lua/test.sh

```diff
@@ -12,7 +12,7 @@ export VIS_THEME=theme
 if [ $# -gt 0 ]; then
 	test_files=$@
 else
-	test_files=$(find . -type f -name "*.in")
+	test_files="$(find . -type f -name "*.in") basic_empty_file.in"
```

Comment:
Just add an empty `basic_empty_file.in` to the repository, i.e. `touch basic_empty_file.in`.

Are you going to update these commits?